### ISSUE TITLE
a11y - site setup - Addon-ons - cms ui

### ISSUE THREAD

**Issue by Wagner3UB** [plone/volto#5142]
**Describe the bug**
- This page contains a series of accordions that are not reachable via keyboard or screen reader.

* after some tests it was discovered that using a screen reader and navigating using ctrl + alt (option) + arrows, it is possible to get to the accordion items

**To Reproduce**

1. Go to "site setup" -> "Addon-ons"
2. Navigate with the keyboard

**Expected behavior**
- All the accordions items need to have tabindex or be naturally found by keyboard navigation.
- The default accordion behavior (open/close) must work
- The accordion content must be reachable by the keyboard
- All items need to have a proper label to identify which will be properly identified
- All items must receive visual focus 

**Screenshots**
<img width="1713" alt="Schermata 2023-08-29 alle 08 42 47" src="https://github.com/plone/volto/assets/60133113/2efe17f4-3cdc-47aa-b6d0-a0eb8727eaad">

**Software (please complete the following information):**
- Volto Version 17.0.0-alpha.19
- Plone Version 6.0.4

**Comment by deodorhunter** [comment]
Work is mostly done, see [#6345](https://github.com/plone/volto/pull/6345) for detailed changelog.
TODO: missing translations due to code splitting/changes/additions to intl messages
